int64 get_image
int64 process_image
int64 adjust_servo
int64 calculate_transforms
int64 publish_transforms
int64 publish_tag_transforms
int64 publish_plain_image
int64 draw_detections
int64 publish_detections_image
int64 spin
int64 total_time
int64 processing_time

time  image_capture_time
int64 last_publish_timing_diagnostics
